disnake/channel.py

```diff
@@ -416,6 +417,11 @@ async def edit(
         default_auto_archive_duration: Optional[Union[:class:`int`, :class:`ThreadArchiveDuration`]]
             The new default auto archive duration in minutes for threads created in this channel.
             Must be one of ``60``, ``1440``, ``4320``, or ``10080``.
+        flags: :class:`ChannelFlags`
+            The new flags to set for this channel.
```

Comment:
This should clarify that all old flags will be overwritten - the current description could be interpreted as setting new flags while keeping existing ones.